tests/_utils/check_ticker_interval.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+
+def check_worker_ticker_interval(log_path, min_interval):
+    """
+    check the time interval between dm-worker updates when the queue is empty
+
+    log_path: dm-worker-log path
+    min_interval: min interval
+    """
+    logs = []
+    with open(log_path, "r") as f:
```

Comment:
use python here because i can't use bash to get last two logs i need, 
i don't know why `head` cmd seems not working  in test scripts but work in my local shell
I spent 4 hours debugging this problem and was frustrated that it didn't work out in the end
so why not use a high-level language to deal with this?

Reply:
it's OK to use python since we have `check_point`.

maybe that's caused by `head` has many distributions, like BSD/GNU